### PR TITLE
test quotes

### PR DESCRIPTION
**Pull request by meek0**

### CODE REVIEW

File: src/main/conf/import.json [hunk 1]
```diff
@@ -13,19 +13,20 @@
       "key": "delim",
       "title": "Delimiter Character",
       "default": ",",
-      "maximum": 1,
+      "maxLength": 1,
```
Comment (ymarcon) [inline src/main/conf/import.json]: How would a tab character be provided? If it's `\t`, that makes 2 chars.

Comment (meek0) [reply src/main/conf/import.json]: if no character is provided to delim, it will fallback on read_table wich will read space characters as delimiters.

Comment (meek0) [reply src/main/conf/import.json]: I'm not sure if read_delim should handle that itslef. If so then I can remove read_table.

Comment (meek0) [reply src/main/conf/import.json]: We could use read_table2 instead because it seems to be more lax than read_table.

Comment (ymarcon) [reply src/main/conf/import.json]: That's not my point. I am just asking how to specify that the delimiter is a tab?

Comment (meek0) [reply src/main/conf/import.json]: no delim character means we use read_table, I changed it to use read_table2 to be more lenient.

Comment (ymarcon) [reply src/main/conf/import.json]: I am not talking about specifying no delimiter, I am talking about a tabulation separated file.

**Pull request merged**